beautyjoy/BJC-Teacher-Tracker
# 187129573 change teacher status

### [Pivotal Tracker Link][tracker]


[tracker]: https://www.pivotaltracker.com/story/show/your-story-id

## What this PR does:


Allows changing a teacher's application status. The frontend was demonstrated at the customer meeting.

This pull request fixes|implements (pick one...) ______.

### Include screenshots, videos, etc.

#### Who authored this PR?

Arush




### How should this PR be tested?

Look at the Cucumber tests. They verify that the feature works as intended.

* Is there a deploy we can view?

Yes: [https://sp24-01-bjc-teachers-d0e972207d82.herokuapp.com/dashboard](https://sp24-01-bjc-teachers-d0e972207d82.herokuapp.com/dashboard)


* What do the specs/features test?
* Are there edge cases to watch out for?

#### Are there any complications to deploying this?



### Checklist:
